scripts/build/package.js

```diff
@@ -13,6 +13,8 @@ const util = require('util');
 const { minify } = require('uglify-js');
 
 const UTF8 = 'utf8';
+// yellow and bold
+const WARN = '\\x1b[33m\\x1b[1mwarn\\x1b[22m\\x1b[39m';
```

Comment:
I'm not sure about this. Is it supposed to work within the browser also? What happens with terminals without colors (see [`NO_COLOR`](https://no-color.org/)? Also, it works under Windows CMD? I think that is better to not include color nor formatting for the warning.

Reply:
I think it'll help people notice that it's a warning.
My browser (Chromium) renders it fine: ![image](https://user-images.githubusercontent.com/58836760/143091296-b920b4c1-05e6-4e40-aa65-b8d9802f4480.png)

It does break on Firefox though, but I don't think that's a big deal:
![image](https://user-images.githubusercontent.com/58836760/143091549-8944fd9b-f208-492f-ab06-f92c44c03fa1.png)

For the `NO_COLOR` part, we could use [`picocolors`](https://github.com/alexeyraspopov/picocolors) (which is what I looked at to get those ansi codes), which detects that. I'm not sure about Windows cmd, but I'm guessing ansi codes don't work there.

Reply:
I think it's not worth to add a dependency to the package for a warning, Firefox needs HTML for formatting and in Windows CMD doesn't work.

Reply:
Actually I just tested in Windows CMD and it works fine. I guess Node handles it somehow.

Reply:
Note that `console.warn` itself renders with yellow background color in browsers.

Reply:
I don't think it's worth it either, and I'll remove the formatting, but that's interesting to know.

Reply:
🤔 Wow that's new, I would swear it hadn't worked long time ago...